### PR TITLE
Redirect to a placeholder URL about this service

### PR DESCRIPTION
**Pull request by dims**
For now redirect to a place holder URL (github repo for this service)

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: cmd/archeio/app/handlers.go [hunk 1]
```diff
@@ -39,6 +44,8 @@ func MakeHandler(upstreamRegistry string) http.Handler {
 		switch {
 		case strings.HasPrefix(path, "/v2/"):
 			doV2(w, r)
+		case path == "/":
```
Comment (BenTheElder) [inline cmd/archeio/app/handlers.go:47]: we should get a unit test case for this

Comment (dims) [reply cmd/archeio/app/handlers.go:47]: https://github.com/kubernetes-sigs/oci-proxy/pull/51/files

File: cmd/archeio/app/handlers.go [hunk 2]
```diff
@@ -26,6 +26,11 @@ import (
 	"sigs.k8s.io/oci-proxy/pkg/net/cidrs/aws"
 )
 
+const (
+	// hard coded for now
+	infoURL = "https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io"
```
Comment (BenTheElder) [inline cmd/archeio/app/handlers.go:31]: I think we should probably switch to https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-(registry.k8s.io)

this link doesn't seem to have much relevant, we're not using the things that are there

Comment (BenTheElder) [reply cmd/archeio/app/handlers.go:31]: will try to file a follow up soon. this just came up after the community update today

**Pull request merged**